scripts/gm/gm.toml

```diff
@@ -1,38 +1,58 @@
 # Example configuration with one validator and one full node
 
-# the global section defines application-wide defaults. All the entries here can be overridden on a per-node basis,
-# except if otherwise noted.
+##
+## Global section defines application-wide defaults.
+##
+
+# All the entries here can be overridden on a per-node basis, except if indicated otherwise.
 [global]
 
-# Path to the `gaiad` binary to use.
+# Path to the `gaiad` (or other SDK-based) binary to use.
```

Comment:
Perhaps we can rename `gm` into `lcm` (local cosmos manager) or something like that later on ;p

Reply:
Open to any naming. :) ("local cosmos", LOL) I kept it two letters for shortness's sake (and to adhere to age-old Unix preferences).

Reply:
"loco" (spanish for crazy) also sounds fun.